CandyMarket/CandyMarket/DataAccess/CandyRepo.cs

```diff
@@ -14,5 +17,45 @@ public CandyRepo(IConfiguration config)
         {
             connectionString = config.GetConnectionString("CandyMarket");
         }
+
+        public IEnumerable<Candy> GetByUserId(int userId)
+        {
+            var sql = @"
+                        select candies.CandyId, candies.CandyName, Candies.Manufacturer, candies.FlavorCategory
```

Comment:
While working on update I got to thinking we may need to add a couple more properties to the candy class, which would affect your select here.  Let's talk about it Sat morning

Reply:
we will use different views to resolve this